engine/src/main/java/org/terasology/audio/AudioManager.java

```diff
@@ -50,12 +50,20 @@
 
     void playMusic(StreamingSound music);
 
+    void playMusic(StreamingSound music, boolean loop);
```

Comment:
Not at all, once I get the bug above figured out I think I'll be able to add some useful javadocs. :)